src/utils/daos_autotest.c

```diff
@@ -156,6 +188,36 @@ ccreate(void)
 		return -1;
 	}
 
+	/** Create container with RF=1 */
+	daos_prop_t	*prop;
+
+	prop = daos_prop_alloc(1);
+	prop->dpp_entries[0].dpe_type = DAOS_PROP_CO_REDUN_FAC;
+	prop->dpp_entries[0].dpe_val = DAOS_PROP_CO_REDUN_RF1;
+
+	uuid_generate(cuuid2);
+
+	rc = daos_cont_create(poh, cuuid2, prop, NULL);
```

Comment:
cuuid2 should just be a string here. e.g. "autotest-$hostname-$pid"

Reply:
Labels are now utilized for containers.